manifests/init.pp

```diff
@@ -71,7 +71,7 @@
   }
 
   # Only debian and ubuntu have this style of defaults for startup.
-  case $facts['osfamily'] {
+  case $facts['os.family'] {
```

Comment:
Dot notation doesn't work
```suggestion
  case $facts['os']['family'] {
```

However, I'm hesitant to start changing these facts and would rather do them wholesale using https://rubygems.org/gems/puppet-lint-legacy_facts-check